app/src/main/res/values-de/help.xml

```diff
@@ -0,0 +1,231 @@
+        <h2>Ziel</h2>
+        Dies ist die öffentliche Adresse des Wallets, zu dem du Moneroj gesendet hast.
+        <h2>Zahlungs-ID</h2>
+        Du kannst eine Zahlungs-ID benutzen um Zahlungen zu identifizieren. Dies ist optional und ist nicht öffentlich einsehbar. 
+		Zum Beispiel kann ein Unternehmen damit Zahlungen und Verkäufe miteinander verbinden.
+        <h2>TX ID</h2>
+        Dies ist deine Transaktions-ID die du benutzen kannst um verschleierte Transaktionen mit einem Monero
+        Blockchain-Explorer wie <a href="https://xmrchain.net/">https://xmrchain.net/</a> zu identifizieren.
+        <h2>TX Schlüssel (Transaktionsschlüssel)</h2>
+        Dies ist dein geheimer Transaktionsschlüssel. Bewahre ihn sicher auf da er Dritten deine Signatur in einem Ring offenbart 
+		und deshalb deine Transaktionen transparent macht.
+        <h2>Block</h2>
+        Dies ist der Block der deine Transaktion beinhaltet..
+    ]]></string>
+
+    <string name="help_send"><![CDATA[
+        <h1>Senden</h1>
+        <h2>Empfängeradresse</h2>
+        <p>Dies ist die öffentliche Adresse des Wallets, an das du Moneroj sendest. Du kannst sie aus der Zwischenablage kopieren, 
+		einen QR-Code scannen oder sie manuell eingeben. Vergewissere dich, dass es die richtige Adresse ist um
+        sicherzustellen die Moneroj nicht irrtümlich an die falsche Adresse zu senden.</p>
+        <p>Zusätzlich zum Senden von XMR kannst du auch BTC mittels des XMR.TO Service versenden (siehe https://xmr.to
+        für Details). Siehe den Abschnitt über das Versenden von BTC unten.</p>
+        <h2>Zahlungs-ID</h2>
+        <p>Du kannst eine Zahlungs-ID benutzen um den Grund einer Transaktion zwischen zwei Parteien zu identifizieren. Dies ist optional 
+		und nicht öffentlich einsehbar. Zum Beispiel kann ein Unternehmen damit Zahlungen und Verkäufe miteinander verbinden.<p>
+        <h2>Ringgröße</h2>
+        <p>Es gibt verschiedene Ringgrößen in Monerujo aus denen du wählen kannst. Falls du ein neuer Nutzer bist
+        empfehlen wir bei der Standard Ringgröße zu bleiben (aktuell 7). Die Ringgröße zu erhöhen erhöht die Anzahl der Signaturen in einer Ring-Signatur, 
+		was theoretisch das Erraten der richtigen Signatur erschwert. Eine höhere Ringgröße lässt eine Transaktion jedoch 
+		auf der Blockchain auch herausstechen.</p>
+        <h2>Priorität</h2>
+        <p>Diese Einstellung bestimmt wie zügig deine Transaktion in einem Block auf der Blockchain verarbeitet wird.
+		Eine höhere Priorität bedeutet eine höhere Transaktionsgebühr, eine niedrige Priorität bedeutet eine 
+		niedrigere Transaktionsgebühr. Bitte bedenke, dass eine Transaktion mit einer niedrigen Priorität unter Umständen
+		eine Weile benötigt bis sie auf der Blockchain verarbeitet wird. Die Standard-Priorität
+        ist \"Mittel\".</p>
+        <h1>BTC Senden</h1>
+        <h2>XMR.TO</h2>
+        <p>XMR.TO ist ein Drittanbieter-Service welcher als Tauschbörse von Monero zu Bitcoin fungiert.
```

Comment:
not sure about "Tauschbörse" because Börse has many players. maybe "Wechselservice"?